cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -595,23 +602,23 @@ you have a use case for it, please reach out to us at `cdap-user@googlegroups.co
 
 #. Stop all CDAP processes::
 
-     for i in `ls /etc/init.d/ | grep cdap` ; do sudo service $i stop ; done
+     $ for i in `ls /etc/init.d/ | grep cdap` ; do sudo service $i stop ; done
 
 #. Update the CDAP packages by running either of these methods:
 
    - Using Yum (on one line)::
 
-       sudo yum install cdap cdap-gateway
-                             cdap-hbase-compat-0.94 cdap-hbase-compat-0.96
-                             cdap-hbase-compat-0.98 cdap-kafka cdap-master
-                             cdap-security cdap-web-app
+       $ sudo yum install cdap cdap-gateway
```

Comment:
We need to avoid displaying commands wrapped onto multiple lines, unless they have '\' at the end of each line that wraps to another. The user should be able to copy (from after command prompt, if applicable) and paste the whole thing into a terminal window and have it run as expected.  This may look 'prettier', but it will not work as it is written here.